src/floating-focus.scss

```diff
@@ -46,3 +46,8 @@
 		border: 0 !important;
 	}
 }
+
+.floating-focus-enabled :focus, .floating-focus-enabled .focus {
```

Comment:
Not sure if we even need this. Maybe just use the default `0`?

Reply:
Okay I actually like that more. That way we also definitely won't get issues with this value overwriting the one the implementing party defined, and if it looks ugly they should just follow the readme and encounter the line where they see you can specify your own outline offset.